app/templates/components/widgets/forms/ticket-input.hbs

```diff
@@ -4,8 +4,8 @@
       {{input type='text' name='ticket_name' placeholder=(t 'Ticket Name') value=ticket.name}}
     </div>
   </div>
-  <div class="column">
-    {{#if (eq ticket.type 'paid')}}
+  <div class="two wide column">
+    {{#if (or ticket.type 'paid' ticket.type 'donation')}}
```

Comment:
Is this a valid expression? You have removed the eq operator?

Reply:
@CosmicCoder96 hmm. weird. it works without/with the `eq` operator.
   `{{#if (or (eq ticket.type 'paid') (eq ticket.type 'donation'))}}`

Reply:
But will update with `eq` though. Don't want to mess up the logic.